stable/database/files/nuosm

```diff
@@ -9,9 +9,9 @@
 
 . ${NUODB_HOME}/etc/nuodb_setup.sh
 
-: ${NUODB_ARCHIVEDIR:=/var/opt/nuodb/archive}
+: ${NUODB_ARCHIVEDIR:=/var/opt/nuodb/archive/${NUODB_DOMAIN}/${DB_NAME}}
 : ${NUODB_BACKUPDIR:=/var/opt/nuodb/backup}
-: ${NUODB_JOURNALDIR:=/var/opt/nuodb/journal}
+: ${NUODB_JOURNALDIR:=/var/opt/nuodb/journal/${NUODB_DOMAIN}/${DB_NAME}}
 : ${NUODB_STORAGE_PASSWORDS_DIR:=/etc/nuodb/tde}
 : ${NUODB_DOMAIN:="nuodb"}
 : ${DB_NAME:="demo"}
```

Comment:
For clarity, you may want to move these two lines before defaulting the `${NUODB_ARCHIVEDIR}`.
It works now because `${NUODB_DOMAIN}` and `${DB_NAME}` are both set in the templates.